roles/cinder-data/tasks/main.yml

```diff
@@ -78,7 +78,7 @@
   systemd_service: name=cinder-volume user=cinder
                    cmd=/usr/local/bin/cinder-volume
                    config_dirs=/etc/cinder
-  when: os == 'rhel' and openstack_install_method == 'package'
+  when: openstack_install_method != 'distro'
```

Comment:
agreed...  this should test for rhel+distro ... as it sits right now it would try and run that on ubuntu.